src/main/resources/api/service_openapi.yaml

```diff
@@ -531,6 +535,11 @@ components:
           description: The ID of the Data Repo snapshot
           type: string
 
+    WorkspaceStageEnumModel:
```

Comment:
`WorkspaceStage` is also fine, or `WorkspaceStageModel`. There shouldn't be too many classes that import both enums, but we at least don't need to name it Enum.